R/organization.R

```diff
@@ -26,6 +26,16 @@ download_organization_repos <- function(organizations = NULL,
   if (!dir.exists(dest_folder)) dir.create(dest_folder)
   withr::local_dir(dest_folder)
 
+  external_repos <- extra_repos[!names(extra_repos) %in% organizations]
+
+  if (length(external_repos) > 0) {
+    warning(
+      "The following repos belong to external organizations and have been ",
+      "dropped: ", paste(external_repos, collapse = ", ")
```

Comment:
```suggestion
      "dropped: ", toString(external_repos)
```

I'll let you find what R version minimal dependency we need to add here :stuck_out_tongue_winking_eye:

Reply:
For sure less than 4.1 so no issue here :smile:

Reply:
But out of curiosity, looks like this function has been here since the beginning: https://github.com/wch/r-source/commit/2d832a89acdd3f656acd38cb4973930500177069